CompanyEmployees.Infrastructure.Persistence/Configurations/AttendanceConfiguration.cs

```diff
@@ -0,0 +1,32 @@
+using CompanyEmployees.Core.Domain.Entities;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace CompanyEmployees.Infrastructure.Persistence.Configurations;
+
+public class AttendanceConfiguration : IEntityTypeConfiguration<Attendance>
+{
+    public void Configure(EntityTypeBuilder<Attendance> builder)
+    {
+        builder.HasKey(a => a.Id);
+
+        // BUG: Navigation property name is wrong - entity has EmployeeDetails not Employee
+        builder.HasOne(a => a.Employee)
```

Comment:
The navigation property `Employee` does not exist on the `Attendance` entity. Based on the comment, it should be `EmployeeDetails`. Using the wrong property will cause runtime mapping errors.

```suggestion
builder.HasOne(a => a.EmployeeDetails)
```